.changeset/config.json

```
@@ -3,7 +3,7 @@
     "changelog": ["@changesets/changelog-github", { "repo": "withastro/adapters" }],
     "commit": false,
     "access": "public",
-    "baseBranch": "main",
+    "baseBranch": "next",
```

This needs to be changed back to `main`
```suggestion
    "baseBranch": "main",
```